### PR TITLE
[ft-render-landing-page 161863306]

### PR DESCRIPTION
**Pull request by ddouglasz**
### What does this PR do?
- Render the landing page.
### Description of Task to be completed?
- The PR converts the template HTML to react component

### How should this be manually tested?
- clone the project.
- cd into client on your terminal
- run `npm run start`

### Any background context you want to provide?
- None

### What are the relevant pivotal tracker stories?

[161863306](https://www.pivotaltracker.com/story/show/161863306)
### Screenshots (if appropriate)
image

### Questions:
- None

### CODE REVIEW

File: client/src/components/landingPage.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+class LandingPage extends Component {
+    render() {
+        return (
+            <div>
+                <div className="header">
+                    <div className="header-cover">
+                        <div className="container">
+                        <NavBar />
+                            {/* <nav className="navbar navbar-expand-lg navbar-dark">
```
Comment (Rhotimee) [inline client/src/components/landingPage.jsx]: Hey @ddouglasz, 
Is there any reason we have this commented code ?

File: client/src/components/landingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                        </div>
+                    </div>
+                </div>
+
+                <div id="section4" className="container-fluid text-center py-5 my-5">
+                    <h3 className="my-3">Lorem ipsum dolor sit.</h3>
+                    <button className="btn btn-lg btn-success donate-button">Donate</button>
+                </div>
+
+                <div id="footer" className="py-5">
```
Comment (Rhotimee) [inline client/src/components/landingPage.jsx]: I feel it will be nice if we can have this footer as a separate component just like the navbar.

Comment (Daramola98) [reply client/src/components/landingPage.jsx]: Yeah  Nice Suggestion @Rhotimee

File: client/package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
+    "node-sass": "^4.10.0",
```
Comment (Daramola98) [inline client/package.json]: Why do we have this package installed, I cannot see it being used an

Comment (ddouglasz) [reply client/package.json]: Noted, All changes would be implemented today.

File: client/src/components/landingPage.jsx (hunk 2 above)
Comment (ddouglasz) [reply client/src/components/landingPage.jsx]: Noted. It would be done today

File: client/src/components/landingPage.jsx (hunk 1 above)
Comment (ddouglasz) [reply client/src/components/landingPage.jsx]: An oversight. I would be corrected today.

**Pull request merged**